vdc/gcp-config.yaml.in

```diff
@@ -80,7 +86,7 @@ resources:
           diskType: pd-standard
```

Comment:
hmm, there's no non-preemptible pool here, notebook will likely be unable to schedule notebook workers.